cli/azd/resources/scaffold/templates/next-steps.mdt

```diff
@@ -12,12 +12,11 @@
 
 ### Define environment variables for running services
 
```

Comment:
Should we have an example here?

Reply:
Does the placeholder example always generated in `main.parameters.json` help? Or do we need a concrete one. Wonder what good variable examples we should use..

Reply:
Sorry - I didn't see that. If we have a commented out block or a placeholder with the right fields, that should work

Reply:
No worries -- just wanted to confirm. Here's the placeholder below. Unfortunately, we're dealing with JSON here so having comments was not quite an option.

![](https://user-images.githubusercontent.com/2322434/275932022-d45ee22d-2c73-4f7b-849c-1d72fbffae68.png)

Reply:
Yeah that seems reasonable to start!